stream/src/main/scala/org/apache/pekko/stream/javadsl/Flow.scala

```diff
@@ -1907,6 +1907,61 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
         case elem if clazz.isInstance(elem) => supplier.get()
       })
 
+  /**
+   * onErrorComplete allows to complete the stream when an upstream error occurs.
+   *
+   * Since the underlying failure signal onError arrives out-of-band, it might jump over existing elements.
+   * This operator can recover the failure signal, but not the skipped elements, which will be dropped.
+   *
+   * '''Emits when''' element is available from the upstream
+   *
+   * '''Backpressures when''' downstream backpressures
+   *
+   * '''Completes when''' upstream completes or failed with exception is an instance of the provided type
+   *
+   * '''Cancels when''' downstream cancels
+   *  @since 1.1.0
+   */
+  def onErrorComplete(): javadsl.Flow[In, Out, Mat] = onErrorComplete(classOf[Throwable])
+
+  /**
+   * onErrorComplete allows to complete the stream when an upstream error occurs.
```

Comment:
a upstream
Same as above for others

Reply:
I am a native English speaker and this is already correct - `an upstream` is correct.

`an` before words beginning with vowels sounds

- `u` is complicated because `u` often has an implied `y` sound at the start of words (eg `Ukraine`) - but `up` has no implied `y` sound - with a `u` that is pronouned `yu` then you do not use `an` - eg "a `u`" is pronounced  "a yu")

Reply:
no y ，Upstream ,^p'stri:m

Reply:
Thank you, I made a mistake on `ʌ`

But this PR has both an upstream and a upstream